.github/workflows/rust.yml

```diff
@@ -34,6 +34,8 @@ jobs:
       run: echo "Some of jobs failed" && false
 
   build_n_test_android:
+    strategy:
```

Comment:
This field is applicable only for matrix builds.
Not needed for single build job.
Ref: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategyfail-fast